Clustering/src/clusterer.cxx

```diff
@@ -24,13 +32,40 @@ void Clusterer::Clustering(const char *inputFile) {
   FileManager inputData(inputFile);
 
   std::unordered_map<int, std::vector<int>> eventIndices;
+  // TTree * tree = new
+  // TTree(root["tree"]["clusterer"]["name"].As<std::string>().c_str(),// for
+  // the future if we will want a configuration file
+  // root["tree"]["clusterer"]["name"].As<std::string>().c_str());
+  TTree *tree = new TTree("Clusters_tree", "Clusters_tree");
+  std::vector<Cluster> clusterVector;
+  std::vector<std::vector<Cluster>> clustersVector;
```

Comment:
Just for future readability, I would call them EventClusters and RunClusters, or something like this to distinguish them

Reply:
ok